Dockerfile

```diff
@@ -1,45 +1,58 @@
-FROM node:16.16.0-alpine3.15 AS base
-
 # Install dependencies only when needed
-FROM base AS deps
+FROM node:16-alpine AS deps
```

Comment:
Should still alias it as `base` so that we only have to change the image in one place when updating it in the future.